### PR TITLE
vita: No more need vita specific codesets

### PR DESCRIPTION
**Pull request by d3m3vilurr**
combine into unix.c
remain problem is that vita system doesn't have the `ioctl`. :)

### CODE REVIEW

**Review by cgutman — COMMENTED** [review]
Can we get rid of some of the definitions in vita.c too? It looks like newlib at least has `TCP_NODELAY` now.

File: unix.c [hunk 1]
```diff
@@ -232,13 +269,18 @@ int
...
     switch (option)
     {
         case ENET_SOCKOPT_NONBLOCK:
 #ifdef HAS_FCNTL
             result = fcntl (socket, F_SETFL, (value ? O_NONBLOCK : 0) | (fcntl (socket, F_GETFL) & ~O_NONBLOCK));
 #else
+#ifdef HAS_IOCTL
             result = ioctl (socket, FIONBIO, & value);
+#else
+            result = setsockopt (socket, SOL_SOCKET, SO_NONBLOCK, (char * ) & _true, sizeof(int));
```
Comment (d3m3vilurr) [inline unix.c]: this just hack & copy from old vita code. I don't remember in the old cases, but it wouldn't work properly with `&value`
(maybe just passing the const number makes problem.)

Comment (cgutman) [reply unix.c]: I don't understand how `&_true` is different than `&value`. Aren't they both int variables set to 1?

Comment (d3m3vilurr) [reply unix.c]: hm. an original code of xyzz was also same. I try to do cross checking in my local.

----

removed it

**Pull request merged**